react/features/toolbox/components/web/Toolbox.js

```diff
@@ -950,6 +957,10 @@ class Toolbox extends Component<Props, State> {
                 key = 'videobackgroundblur'
                 showLabel = { true }
                 visible = { this._shouldShowButton('videobackgroundblur') && !_screensharing } />,
+            <VideoCropPersonButton
+                key = 'videocropforeground'
+                showLabel = { true }
+                visible = { this._shouldShowButton('videocropforeground') && _screensharing && _isPresenter } />,
```

Comment:
Everything except `this._shouldShowButton('videocropforeground')` should be handled by the button itself, internally.